samples/python/llm/benchmark.py

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import argparse
+import time
+from pathlib import Path
+import utils.ov_utils
+import utils.pt_utils
+import utils.model_utils
+import torch
+import numpy as np
+from openvino.runtime import get_version
+from utils.config_class import DEFAULT_MODEL_CLASSES
+import PIL
+import hashlib
+import utils.metrics_print
+import utils.output_csv
+import utils.hook_forward
+import utils.hook_transformers
+import traceback
+from transformers import set_seed
+from PIL import Image
+from utils.memory_profile import MemConsumption
+
+HOOK_UTILS = {"pt": utils.hook_transformers, "ov": utils.hook_forward}
+FW_UTILS = {"pt": utils.pt_utils, "ov": utils.ov_utils}
+DEFAULT_INFERENCE_STEPS = 20
+DEFAULT_SUPER_RESOLUTION_STEPS = 50
+DEFAULT_OUTPUT_TOKEN_SIZE = 512
+MAX_OUTPUT_TOKEN_SIZE = 64 * 1024
+
+memConsumption = MemConsumption()
+
+def gen_iterate_data(iter_idx="", in_size="", infer_count="", out_size="", gen_time="", latency="", res_md5="", max_rss_mem = "", max_shared_mem = "", prompt_idx = ""):
+    iter_data = {}
+    iter_data['iteration'] = iter_idx
+    iter_data['input_size'] = in_size
+    iter_data['infer_count'] = infer_count
+    iter_data['output_size'] = out_size
+    iter_data['generation_time'] = gen_time
+    iter_data['latency'] = latency
+    iter_data['result_md5'] = res_md5
+    iter_data['first_token_latency'] = ""
+    iter_data['other_tokens_avg_latency'] = ""
+    iter_data['max_rss_mem_consumption'] = max_rss_mem
+    iter_data['max_shared_mem_consumption'] = max_shared_mem
+    iter_data['prompt_idx'] = prompt_idx
+    return iter_data
+
+def run_text_generation(input_text, num, model, tokenizer, args, iter_data_list, prompt_index, bench_hook):
+    set_seed(args['seed'])
+    input_text_list = [input_text] * args['batch_size']
+    print(f"input_text={input_text}")
+    input_data = tokenizer(input_text_list, return_tensors="pt")
+    input_data.pop("token_type_ids", None)
+    print("Remove `token_type_ids` from inputs.")
```

Comment:
As the discussion: update this to code comment instead of log

Reply:
Done.